qemu/tests/cfg/nic_teaming.cfg

```diff
@@ -1,6 +1,7 @@
 - nic_teaming: 
     virt_test_type = qemu
     only Linux
+    only bridge
```

Comment:
OVS is also a bridge.

Reply:
Sorry, I mistake it. So do you know any way to limit it to be tested only on the switch bridge? Thanks in advace.

Reply:
I have resume this  file, it will be fixed other way in the future.

Reply:
You can check if it's a Linux bridge or an OVS bridge.